Android/JazzyViewPager/nuget/Xamarin.JazzyViewPager.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Xamarin.JazzyViewPager</id>
+    <title>JazzyViewPager for Xamarin.MonoAndroid44</title>
+    <version>1.0.1</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>An easy to use ViewPager that adds an awesome set of custom swiping animations.</description>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <projectUrl>https://go.microsoft.com/fwlink/?linkid=871870</projectUrl>
+    <licenseUrl>https://go.microsoft.com/fwlink/?linkid=871869</licenseUrl>
+  </metadata>
+  <files>
+    <file src="output/JazzyViewPager.dll" target="lib/Xamarin.Android" />
```

Comment:
This is the wrong target name - to should be MonoAndroid60 or MonoAndroid80, what ever the minimum version of android is